docs/gitbook/examples/initialize.md

```diff
@@ -51,12 +57,12 @@ npm i --save interbit
 The package comes with a set of granular functions for starting a node as well as a handy dandy [deploy](../reference/interbit-cli/deploy.md) function that does all of the work of applying covenants, starting and loading chains, and creating an interbit instance and cli to interact with.
```

Comment:
Trailing backslash in `"deploy": "interbit deploy --manifest interbit.manifest.json"\` causes weird formatting.